slurm-cookbook/docker/README.md

```diff
@@ -0,0 +1,66 @@
+# Example: HPC Toolkit Support Slurm and Docker 
+
+## Overview
+
+This example creates a Slurm Cluster and allows Docker files to run.
+All other components required to support the Slurm cluster are minimally created
+as well: VPC; subnetwork; firewall rules; service accounts.
+
+# Install HPC Toolkit
+
+Follow [the instructions here](https://github.com/GoogleCloudPlatform/hpc-toolkit#quickstart) to install the HPC Toolkit.
+In the Hpc Toolkit folder run an install.
+```
+make install
+```
+
+# Run the HPC Toolkit
+
+* Create a GCP Project [as described here](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+ [![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://shell.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/scientific-computing-examples.git)
+
+```
+cd slurm-cookbook/docker
+ghpc slurm-docker.yaml
+```
+
+* Edit the `slurm-docker.yaml` file.
+  * Change `<your project>` to be the `my-project-id` of the project you created in the previous step
+  * Change `<oslogin_user_id> to your OS Login id, which should follow: `USERNAME_DOMAIN_SUFFIX`, with all punctuation replaced by underscore, `_`.
```

Comment:
USERNAME has to be prefixed with 'ext'

Reply:
I don't see that ext is required when I login. I confirmed here: https://cloud.google.com/compute/docs/oslogin/set-up-oslogin#step_3_connect_to_vms

Is there a counter reference doc?

Reply:
None of this is documented as far as I know. You may not be getting the `ext_` prefix if you use Cloud Shell. I always use my Cloudtop. I always get the `ext_` prefix.

Reply:
I have reproduced this. I get the "ext_" when I connect my google.com ldap to an external project, e.g. gcp.solutions domain. Can you check if your non-google ID has the prefix?